### ISSUE TITLE
[Exasol] Connection invalidation doesn't read schema list for SQL Editor

### ISSUE THREAD

**Issue by Matvey16** [dbeaver/dbeaver#14069]
#### System information: 
- Operating system (distribution) and version
- DBeaver version 21.2.2

#### Connection specification:
- Database name and version Exasol

#### Describe the problem you're observing:
If the connection has been lost (for example due to inactivity), schema list can't be shown due to lost connection. But Invalidating the connection doesn't help and just disables the error message. Schema list is still blank

#### Steps to reproduce, if exist:
1. Wait for a connection to get disconnected
2. Click `Invalidate/Reconnect`
3. Open schema list in SQL Editor

**Expected:**  Schema list shows schemas
**Actual:**  Schema list is blank

![KI5tFcdA1T](https://user-images.githubusercontent.com/82543000/135461900-849c83e6-565b-458d-b8c2-ac0c649750b3.gif)

**Comment by Sargul** [comment]
Hi,

I was not able to reproduce the issue.

Steps I tried:
* Open Connection
* Open Schema List (as shown in your screenshot)
* Restart the Exasol Database (to generate error)

However as soon as you open "Schemas" in the connection the cache for the schema selector is refreshed and available and no access to the database is done when opening the dropdown to change schema.

I also tried these steps:
* Open Connection
* Restart DB to generate error
* Choose dropdown -> empty
* Invalidate/Reconnect
* Schema dropdown -> list is filled

Perhaps you didn't wait for the reconnect to complete?

Check the messages in the query manager to see if reconnect finished
![reproduce](https://user-images.githubusercontent.com/5595511/137017183-d606e343-b4cd-4253-8ea4-b9e4fec0540f.gif)
